client/my-sites/beta-testing/main.jsx

```diff
@@ -0,0 +1,183 @@
+/**
+ * External dependencies
+ */
+import React, { Fragment } from 'react';
+import { connect } from 'react-redux';
+import { useTranslate } from 'i18n-calypso';
+import FollowButtonContainer from 'calypso/blocks/follow-button';
+import Gridicon from 'calypso/components/gridicon';
+import { useQuery } from 'react-query';
```

Comment:
@claudiucelfilip I noticed that the other instances of `useQuery` in this project are abstracted out and live in the `data` directory. Do we want to follow that same pattern here for consistency?

FWIW, this is my first look at `react-query` so I may be missing something - an initial observation, really! 😄

Reply:
That's good point, thanks! 
I added a more general posts query to the `data` folder, which can be used in other locations.